### ISSUE TITLE
Can't build - can't find reason for requirement on github.com/google/pprof@v0.0.0-20210407192527-94a9f03dee38

### ISSUE THREAD

**Issue by heapdavid** [greenpau/caddy-security#217]
**Describe the issue**

Building Caddy 2.6.4 with caddy-security fails with 
```
...
#7 29.48 go: downloading github.com/jackc/pgpassfile v1.0.0
#7 29.48 go: downloading github.com/jackc/pgservicefile v0.0.0-20200714003250-2b9c44734f2b
#7 29.49 go: downloading github.com/mgutz/ansi v0.0.0-20200706080929-d51e80ef957d
#7 29.51 go: downloading github.com/shurcooL/sanitized_anchor_name v1.0.0
#7 29.54 go: downloading github.com/mattn/go-colorable v0.1.8
#7 29.59 go: downloading github.com/mattn/go-isatty v0.0.13
#7 33.20 go: downloading github.com/antlr/antlr4 v4.11.1+incompatible
#7 35.69 panic: internal error: can't find reason for requirement on github.com/google/pprof@v0.0.0-20210407192527-94a9f03dee38
#7 35.69 
```
Looks to be related to known upstream Go error:

- https://github.com/caddyserver/caddy/issues/5301
- https://github.com/caddy-dns/netcup/issues/1

**Expected behavior**

Build completes successfully

**Comment by heapdavid** [comment]
Suggested fix is for 2.6.3 in this thread but hoping it works for 2.6.4 (although the workaround below doesn't seem to) https://github.com/caddyserver/caddy/issues/5301#issuecomment-1425404341

**Comment by heapdavid** [comment]
Sorry ignore me, there was another module in the xcaddy build arguments that was causing this
